lib/galaxy/tools/__init__.py

```diff
@@ -1019,7 +1019,7 @@ def parse(self, tool_source: ToolSource, guid=None, dynamic=False):
         self.python_template_version = tool_source.parse_python_template_version()
```

Comment:
Is there a use case for keeping `float` as the type of `Tool.profile` ? It seems it should be easier to make it a `packaging.version.Version` directly at line 1003 above.

Reply:
One problem is that `str(Version("21.09"))` is `"21.9"` which is at least an annoyance for the tool linter output, i.e. the linter would output that the tool specifies 21.9 even if the user typed 21.09. 

So I think it might be better (and easier) to stick with `float`.